index.bs

```diff
@@ -5116,7 +5206,7 @@ which is <a>a-rate</a> when any connected {{PannerNode}} is
 			min-notes: Approximately -3.4028235e38
 			max: <a>most-positive-single-float</a>
 			max-notes: Approximately 3.4028235e38
-			rate:
+			rate: "{{AutomationRate/a-rate}}"
```

Comment:
Where is this in the rendered page? I can't find these.

Reply:
Yeah, the online converter doesn't know that it needs to include some macros, so these don't show up anywhere.  You'll have to run bikeshed locally for this to work.

The alternative is for me to generate and check-in index.html so it can be viewed.  I'll do that.